de.dlr.sc.overtarget.language/src/de/dlr/sc/overtarget/language/TargetModel.xtext

```diff
@@ -84,7 +84,7 @@ UrlElementVariable:
 ;
 
 RepositoryLocation:
-	'RepositoryLocation' name=ID 'url' url=UrlExpression '{'
+	(referenceTarget?='ReferenceTarget')? 'RepositoryLocation' name=ID 'url' url=UrlExpression '{'
```

Comment:
I guess we should document that decently.... Otherwise no one will understand what the ReferenceTarget keyword does. Maybe we have to provide an example in the documentation or so,,,

Reply:
Actually is "ReferencedTarget" maybe better? Its not the location of the reference itself but rather the location of the referenced target model